### PR TITLE
Python CI

### PR DESCRIPTION
**Pull request by lukesneeringer**
This pull request adds CI for Python.

*In its current state, the actual tests fail*, because the existing code in master right now has failing tests. I have not determined yet what the correct process for addressing that is (merge this first, then merge the fix to prove that CI shows fixed, or the other way around). I will work through that tomorrow.

### CODE REVIEW

**Comment by lukesneeringer** [comment]
Or the tests just will not trigger. Grumble.

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,6 +6,75 @@ jobs:
...
+          command: |
+            touch trigger_job.sh && chmod a+x trigger_job.sh
+            echo >> trigger_job.sh <<EOF
+            #!/bin/bash
+            # More information about this script: http://bit.ly/2pn9lX8
+            job_name=$1
+            curl -u ${CIRCLE_API_TOKEN}: \
+              -d build_parameters[CIRCLE_JOB]=${job_name} \
+              -d revision=$CIRCLE_SHA1 \
+              https://circleci.com/api/v1.1/project/github/your_org/your_project/tree/$CIRCLE_BRANCH
```
Comment (Dominik-K) [inline .circleci/config.yml]: replacing `your_org/your_project` should fix it:
`https://circleci.com/api/v1.1/project/github/$CIRCLE_PROJECT_USERNAME/CIRCLE_PROJECT_REPONAME/tree/$CIRCLE_BRANCH`

Comment (lukesneeringer) [inline .circleci/config.yml]: Yeah, I just caught that. I am a moron. :-) Thanks.

**Comment by lukesneeringer** [comment]
For reasons that should have been obvious to me, `$CIRCLE_API_TOKEN` is not set on forks. Closing this PR and making a new one on the base repo.

**Pull request closed without merging**